tcms/testplans/static/testplans/js/mutable.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(() => {
+    loadProductVersions();
+});
+
+$('#id_product').change(() => loadProductVersions());
+
+const loadProductVersions = () => {
```

Comment:
should I just copy and paste it. I strived to write a brand new js file, without copying from the other 1500 lines long monstrosity. I think as this evolves we will write more functions and then when we encounter duplication will refactor, rather than copy the old logic, which is very generic, but hardly readable

Reply:
> should I just copy and paste it.

hell no. Include the other .js file in the HTML before the current one and the function will become available for your onchange handler.

Reply:
Sure, I could do that. But wasn't the purpose here to extract some of the old js code. This way we will see for possible duplicated and/or unused code. Just importing the old one would couple us to it and make the situation worse for the future

Reply:
This is exactly what we are doing. If you do `git log` & `git blame` on `tcms/static/js/utils.js` you will see that it is a new file and `update_version_select_from_product()`  is a callback intended to be used with the JSON RPC helper.

But we don't want to copy & paste the same function into another file which is one of the original problems that we are solving but instead we want to re-use the existing functions when these functions are good.